cmd/resource/status/cmd/util.go

```diff
@@ -22,23 +25,23 @@ func init() {
 	_ = clientgoscheme.AddToScheme(scheme)
 }
 
-type createClientFunc func() (client.Reader, error)
+type newResolverFunc func(pollInterval time.Duration) (*wait.Resolver, error)
 
 // createClient returns a client for talking to a Kubernetes cluster. The client
```

Comment:
Please update comment to reflect method name change.

Reply:
Done